### DISCUSSION TITLE
GLIBC_2.32' not found (required by /usr/lib/aarch64-linux-gnu/gstreamer-1.0/libgstnvarguscamerasrc.so)

### DISCUSSION THREAD

**Discussion by paroque28** [OE4T/meta-tegra#1623] [General]
I recently got this error while trying to run nvarguscamerasrc from inside the l4t-base container (Ubuntu 18.04)

```(gst-inspect-1.0:107406): GStreamer-WARNING **: 15:14:33.571: Failed to load plugin '/usr/lib/aarch64-linux-gnu/gstreamer-1.0/libgstnvarguscamerasrc.so': /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /usr/lib/aarch64-linux-gnu/gstreamer-1.0/libgstnvarguscamerasrc.so)```

My workaround was installing a newer version of gcc in the container, but do you guys have a better idea?

For context, I am running the latest from kirkstone-r32.7.x (32.7.5)

**Comment by dwalkes** [comment]
Hey @paroque28 
I think the ideal fix for this is a backport of https://github.com/OE4T/meta-tegra/pull/1194

I've started this in https://github.com/Trellis-Logic/meta-tegra/pull/1 but it needs some help.

**Reply by paroque28** [reply]
Alright, let me put some effort there and help you with the backport

**Reply by graugans** [reply]
Just for reference: https://github.com/OE4T/meta-tegra/pull/1638
